tests/integration/sugar/test_wallet.py

```diff
@@ -29,6 +33,56 @@ async def test_generate_faucet_wallet_rel_sub(self, client):
         )
         self.assertTrue(response.is_successful())
 
+    async def test_generate_faucet_wallet_custom_host_async_websockets(self):
```

Comment:
it was going to be a big change to let our test decorator handle custom URIs. For this it seemed easier to just specify the 4 tests by hand.